src/components/SpotlightAnimation/types.ts

```diff
@@ -0,0 +1 @@
+export type AS = Extract<keyof JSX.IntrinsicElements, 'p' | 'span' | 'h1' | 'h2' | 'h3'>
```

Comment:
@paulpopus Why did you name this type `AS` ? Should be something that makes sense so I can read the type and infer what the type probably is

Reply:
renamed it to AllowedElements